algorithm/sorting-algo-python/insertion_sort.py

```diff
@@ -1,17 +1,17 @@
-def insertion_sort(arr):
-    # Traverse through 1 to len(arr)
-    for i in range(1, len(arr)):
-        key = arr[i]
-        # Move elements of arr[0..i-1], that are greater than key,
-        # to one position ahead of their current position
-        j = i - 1
-        while j >= 0 and key < arr[j]:
-            arr[j + 1] = arr[j]
-            j -= 1
-        arr[j + 1] = key
-
-
-# Driver code to test above
-    arr = [12, 11, 13, 5, 6]
-    insertion_sort(arr)
-    print("Sorted array is:", arr)
+def insertion_sort(arr):
+    # Traverse through 1 to len(arr)
+    for i in range(1, len(arr)):
+        key = arr[i]
+        # Move elements of arr[0..i-1], that are greater than key,
+        # to one position ahead of their current position
+        j = i - 1
+        while j >= 0 and key < arr[j]:
+            arr[j + 1] = arr[j]
+            j -= 1
+        arr[j + 1] = key
+
+
+# Driver code to test above
+    arr = [12, 11, 13, 5, 6]
+    insertion_sort(arr)
+    print("Sorted array is:", arr)
```

Comment:
```suggestion
# Driver code to test above
arr = [12, 11, 13, 5, 6]
insertion_sort(arr)
print("Sorted array is:", arr)
```

Reply:
Yes. But those these files are not my code. They already present in the repo. Because of this files, I got conflicts when merging. If you @FerdausPolok  are ok with it, I will update these three files with my working code?

Reply:
Yes @vigneshs-dev. Just wanted to make sure they got fixed with this open PR. No worries, I fixed them with the last commit and now I'm going to merge it. Thanks for your contributions here. 🎉